cinch/roles/jenkins_master/tasks/configure.yml

```diff
@@ -23,5 +23,18 @@
     group: jenkins
     state: directory
 
+- name: set hostname to match inventory hostname
+  hostname:
+    name: "{{ inventory_hostname }}"
+  when: jenkins_ssl_cert is not defined
```

Comment:
I'm confused what this task is doing and why. "inventory_hostname" is a completely user-friendly value and shouldn't be relied on for anything technical if this is being used for things like Host: header matching. Can you explain what this is for?